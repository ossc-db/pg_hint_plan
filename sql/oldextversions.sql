--
-- tests for upgrade paths
--

CREATE EXTENSION pg_hint_plan VERSION "1.3.0";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.1";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.2";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.3";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.4";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.5";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.6";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.7";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.8";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.9";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.10";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.11";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.4";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.4.1";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.4.2";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.4.3";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.4.4";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.5";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.5.1";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.5.2";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.5.3";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.6.0";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.6.1";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.6.2";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.7.0";
\dx+ pg_hint_plan
\d hint_plan.hints
ALTER EXTENSION pg_hint_plan UPDATE TO "1.7.1";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.8.0";
\dx+ pg_hint_plan
ALTER EXTENSION pg_hint_plan UPDATE TO "1.9.0";
\dx+ pg_hint_plan
DROP EXTENSION pg_hint_plan;
