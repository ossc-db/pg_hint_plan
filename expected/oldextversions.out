--
-- tests for upgrade paths
--
CREATE EXTENSION pg_hint_plan VERSION "1.3.0";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.1";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.2";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.3";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.4";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.5";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.6";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.7";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.8";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.3.9";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.4";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.4.1";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.4.2";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.5";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.5.1";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.6.0";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

ALTER EXTENSION pg_hint_plan UPDATE TO "1.7.0";
\dx+ pg_hint_plan
Objects in extension "pg_hint_plan"
       Object description        
---------------------------------
 sequence hint_plan.hints_id_seq
 table hint_plan.hints
 type hint_plan.hints
 type hint_plan.hints[]
(4 rows)

\d hint_plan.hints
                                     Table "hint_plan.hints"
      Column      |  Type   | Collation | Nullable |                   Default                   
------------------+---------+-----------+----------+---------------------------------------------
 id               | integer |           | not null | nextval('hint_plan.hints_id_seq'::regclass)
 query_id         | bigint  |           | not null | 
 application_name | text    |           | not null | 
 hints            | text    |           | not null | 
Indexes:
    "hints_pkey" PRIMARY KEY, btree (id)
    "hints_norm_and_app" UNIQUE, btree (query_id, application_name)

DROP EXTENSION pg_hint_plan;
